sno/working_copy/postgis.py

```diff
@@ -33,54 +32,16 @@
 L = logging.getLogger("sno.working_copy.postgis")
 
 
-def _adapt_geometry_to_db(g):
-    return Binary(g.to_ewkb())
-
-
-register_adapter(Geometry, _adapt_geometry_to_db)
-
-
-def adapt_geometry_from_db(g, dbcur):
-    return Geometry.from_hex_ewkb(g)
-
-
-def adapt_timestamp_from_db(t, dbcur):
-    # TODO - revisit timezones.
-    if t is None:
-        return t
-    # Output timestamps in the same variant of ISO 8601 required by GPKG.
-    return str(t).replace(" ", "T").replace("+00", "Z")
-
-
-def adapt_to_string(v, dbcur):
-    return str(v) if v is not None else None
-
-
-# See https://github.com/psycopg/psycopg2/blob/master/psycopg/typecast_builtins.c
-TIMESTAMP_OID = 1114
-TIMESTAMP = new_type((TIMESTAMP_OID,), "TIMESTAMP", adapt_timestamp_from_db)
-psycopg2.extensions.register_type(TIMESTAMP)
-
-TIMESTAMPTZ_OID = 1184
-TIMESTAMPTZ = new_type((TIMESTAMPTZ_OID,), "TIMESTAMPTZ", adapt_timestamp_from_db)
-psycopg2.extensions.register_type(TIMESTAMPTZ)
-
-
-# We mostly want data out of the DB as strings, just as happens in GPKG.
-# Then we can serialise it using MessagePack.
-# TODO - maybe move adapt-to-string logic to MessagePack?
-ADAPT_TO_STR_TYPES = {
-    1082: "DATE",
-    1083: "TIME",
-    1266: "TIME",
-    704: "INTERVAL",
-    1186: "INTERVAL",
-    1700: "DECIMAL",
-}
-
-for oid in ADAPT_TO_STR_TYPES:
-    t = new_type((oid,), ADAPT_TO_STR_TYPES[oid], adapt_to_string)
-    psycopg2.extensions.register_type(t)
+def insert_or_replace_command(table_name, col_names, schema=None, pk_col_names=None):
```

Comment:
Maybe - I'll try it out. The "on_conflict()" functionality that I'm using is part of the sqlalchemy code for the Postgres dialact, but it will probably work regardless.

This code has moved around a bit as I've started to create table definitions ahead of time, instead of creating them on the fly when they are needed.